package proto_ff;

import "yd_fieldoptions.proto";


message E_VipVip
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_level = 2[(yd_fieldoptions.field_cname) = "vip等级"];
	optional int32 m_exp = 3[(yd_fieldoptions.field_cname) = "经验值"];
	optional int32 m_privilege_group = 4[(yd_fieldoptions.field_cname) = "特权组"];
}


message Sheet_VipVip
{
	repeated E_VipVip E_VipVip_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_VipPrivilege
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 m_group = 2[(yd_fieldoptions.field_cname) = "group"];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "特权类型"];
	optional int32 m_arg = 4[(yd_fieldoptions.field_cname) = "功能参数"];
	optional string m_args = 5[(yd_fieldoptions.field_cname) = "整数功能参数", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_VipPrivilege
{
	repeated E_VipPrivilege E_VipPrivilege_List = 1[(yd_fieldoptions.field_arysize)=60];
}
